install.sh

```diff
@@ -98,7 +98,9 @@ mkdir -p $STATIC_PATH
 cp $SOURCE_PATH/files/rw_test_audio1.wav $MEDIA_PATH
 
 # Copy default production settings file
+mkdir $WWW_PATH/settings
 cp $SOURCE_PATH/files/var-www-roundware-settings.py $WWW_PATH/settings/roundware_production.py
+chown $USERNAME:$USERNAME -R $WWW_PATH/settings
```

Comment:
Hmm... this line is redundant due to: https://github.com/roundware/roundware-server/blob/c0d5cf6f7aef56172b3abb0837997a4a1b89fd79/deploy.sh#L57